scripts/actions/setup-env.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
```

Comment:
more flags?

Reply:
Thanks, yes. First I'll see if I can rewrite it in bb :)

Reply:
Well, there was a some bootstrapping issues between this script setting up the local binary path that `bb` is installed to, but I managed to rewrite it in `bb`.